workflow/usage.rst

```diff
@@ -0,0 +1,104 @@
+.. index::
+    single: Workflow; Usage
+
+How to Use the Workflow
+=======================
+
+The workflow component gives you an object oriented way to work with state
```

Comment:
I'm not sure we can say that, because @lyrixx still continue to say "this is not a *** ***ing state machine" ( ^^ ) even if I have difficulties to explain myself why ...